python/tests/unit/connectors/anthropic/services/test_anthropic_chat_completion.py

```diff
@@ -26,43 +42,120 @@ def mock_settings() -> AnthropicChatPromptExecutionSettings:
 @pytest.fixture
 def mock_anthropic_client_completion() -> AsyncAnthropic:
     client = MagicMock(spec=AsyncAnthropic)
+
     chat_completion_response = AsyncMock()
-    
-    content = [MagicMock(finish_reason="stop", message=MagicMock(role="assistant", content="Test"))]
-    chat_completion_response.content = content
+    chat_completion_response.content = [TextBlock(text="Hello! It's nice to meet you.", type="text")]
+    chat_completion_response.id = "msg_018Vk4Z8oCMoh7sp2TrQGJ7q"
```

Comment:
let's use some kind of `test_id` value for this...

Reply:
fixed, thanks!